hazelcast/src/test/java/com/hazelcast/jet/core/SplitBrainTest.java

```diff
@@ -90,7 +87,7 @@ public void when_quorumIsLostOnMinority_then_jobDoesNotRestartOnMinorityAndCance
             assertOpenEventually(NoOutputSourceP.executionStarted);
         };
 
-        Future[] minorityJobFutureRef = new Future[1];
+        AtomicReference<Future<?>> minorityJobFutureRef = new AtomicReference<>();
```

Comment:
nit: `AtomicReference<Future>` is like sausage with jam - why would you ever do that? I understand why it is here but that is a sign of a convoluted test logic.

Reply:
Just a variable modification inside lambda. This way at least does not cause compiler warnings ;)